runtime/permissions/prompter.rs

```diff
@@ -354,6 +364,20 @@ impl PermissionPrompter for TtyPrompter {
         )
         .unwrap();
       }
+      if let Some(stack) = stack {
+        let len = stack.len();
+        for (idx, frame) in stack.into_iter().enumerate() {
+          writeln!(
+            &mut output,
+            "┃  {} {}",
+            colors::gray(if idx != len - 1 { "├─" } else { "└─" }),
+            colors::gray(deno_core::error::format_frame::<
+              deno_core::error::NoAnsiColors,
+            >(&frame))
+          )
+          .unwrap();
+        }
+      }
```

Comment:
Also if there's no stack trace we should print a line saying how it can be enabled

Reply:
done, however find it a bit odd now that it mentions the env var yet the handling of the env var is done in cli and  not permissions, but thats minor